packages/ketcher-core/src/domain/serializers/ket/schema.json

```diff
@@ -798,38 +781,43 @@
         }
       }
     },
-    "rasterImage": {
+    "image": {
       "type": "object",
-      "required": ["type", "data"],
+      "required": ["type", "format", "boundingBox", "data"],
       "properties": {
         "type": {
-          "const": "rasterImage"
+          "const": "image"
         },
-        "center": {
-          "$ref": "#/definitions/2dCoordinates"
+        "format": {
+          "type": "string",
+          "pattern": "^image/(png|svg\\+xml)"
         },
-        "data": {
+        "boundingBox": {
           "type": "object",
-          "required": ["bitmap", "halfSize"],
+          "required": ["width", "height", "x", "y"],
           "properties": {
-            "bitmap": {
-              "$ref": "#/definitions/imageFile"
+            "width": {
+              "type": "number",
+              "exclusiveMinimum": 0
             },
-            "halfSize": {
-              "type": "object",
-              "required": ["x", "y"],
-              "properties": {
-                "x": {
-                  "type": "number",
-                  "exclusiveMinimum": 0
-                },
-                "y": {
-                  "type": "number",
-                  "exclusiveMinimum": 0
-                }
-              }
+            "height": {
+              "type": "number",
+              "exclusiveMinimum": 0
+            },
+            "x": {
+              "type": "number"
+            },
+            "y": {
+              "type": "number"
+            },
+            "z": {
+              "type": "number"
```

Comment:
Do we need z coordinate?

Reply:
I've checked schema.json file - a lot of entities are using it.
I suppose we can omit it but all the code across the deserialization logic  have it.
We do not require it since it's not in the array of required fields, but if it's present it should be number - not string or object